{{ cookiecutter.app_name }}/setup.py

```diff
@@ -47,17 +47,20 @@
         # Desktop/laptop deployments
         'macos': {
             'app_requires': [{% if cookiecutter.gui_framework == 'Toga' %}
-                'toga-cocoa==0.3.0.dev11',{% endif %}
+                'toga-cocoa==0.3.0.dev11',{% elif cookiecutter.gui_framework == 'PySide2' %}
+                'pyside2==5.13.0',{% endif %}
```

Comment:
Since pyside uses the same PyPI package for every platform, it could be added to `install_requires`, rather than the platform-specific requirements.

Reply:
Fixed in 8829da250c392c0b57d95e0365f0dcaf3ede5532